ui/apps/platform/src/Containers/Collections/CollectionResults.tsx

```diff
@@ -34,6 +36,39 @@ function fetchMatchingDeployments(
     return request;
 }
 
+function RefreshingDeployment() {
```

Comment:
Another possible name to consider `DeploymentSkeleton` contrasts Deployment**This** / Deployment**That**